<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MountControlPanel</class>
 <widget class="QDialog" name="MountControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>326</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Mount Control Panel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="Ekos::MountMotionWidget" name="mountMotion" native="true"/>
   </item>
   <item>
    <widget class="Ekos::MountPositionWidget" name="mountPosition" native="true"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="targetLayout" stretch="0,0">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Target</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Ekos::MountTargetWidget" name="mountTarget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="unparkButtonObject">
       <property name="text">
        <string>UNPARK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="parkButtonObject">
       <property name="text">
        <string>PARK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="statusTextObject">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Status: Idle</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="centerButtonObject">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="zoom-fit-best">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ekos::MountMotionWidget</class>
   <extends>QWidget</extends>
   <header>ekos/mount/mountmotionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Ekos::MountPositionWidget</class>
   <extends>QWidget</extends>
   <header>ekos/mount/mountpositionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Ekos::MountTargetWidget</class>
   <extends>QWidget</extends>
   <header>ekos/mount/mounttargetwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
